src/Discord.Net.Commands/CommandService.cs

```diff
@@ -18,7 +18,7 @@ public class CommandService
         private readonly ConcurrentDictionary<Type, ConcurrentDictionary<Type, TypeReader>> _typeReaders;
         private readonly ConcurrentDictionary<Type, TypeReader> _defaultTypeReaders;
         private readonly ImmutableList<Tuple<Type, Type>> _entityTypeReaders; //TODO: Candidate for C#7 Tuple
-        private readonly ConcurrentBag<ModuleInfo> _moduleDefs;
+        private readonly HashSet<ModuleInfo> _moduleDefs;
```

Comment:
I knew I missed *something* out - I wasn't able to test the code properly due to msbuild complaining about missing targets